tests/unit/web/test_session.py

```
@@ -0,0 +1,98 @@
+        assert len(sid) > 0
+
+    def test_get_returns_session(self) -> None:
+        mgr = SessionManager(ttl_seconds=3600)
+        sid = mgr.create(Path("/tmp/test.pdf"))
+        session = mgr.get(sid)
+        assert session is not None
+        assert session.session_id == sid
+
+    def test_get_expired_returns_none(self) -> None:
+        mgr = SessionManager(ttl_seconds=1)
+        sid = mgr.create(Path("/tmp/test.pdf"))
```

Ruff (0.15.18)

[error] Probable insecure usage of temporary file or directory: "/tmp/test.pdf"

(S108)